src/scenes/home/profile/profile.js

```diff
@@ -53,6 +53,12 @@ const Profile = ({ verified }) => (
           fontAwesomeIcon="faUser"
           url="/signup-info"
         />
+
+        <IconCard
+          title="Our Programs"
+          fontAwesomeIcon="faHandshake"
```

Comment:
As part of merge commit b7468b87, I went ahead and upgraded this icon to the handshake icon [originally intended](https://github.com/OperationCode/operationcode_frontend/pull/855#issuecomment-366135865) for PR #855  - we're now able to use it w/ FontAwesome v5.